src/components/projects/project_item/project_item.gd

```diff
@@ -282,8 +282,8 @@ func _on_run_with_editor(item, editor_flag, action_name, ok_button_text, auto_cl
 	confirmation_dialog.popup_centered()
 	
 
-func _run_with_editor(item, editor_flag, auto_close):
-	item.as_process([editor_flag]).create_process()
+func _run_with_editor(item: Projects.Item, editor_flag, auto_close):
+	item.edit([editor_flag])
```

Comment:
given: take into account the proposed methods project.edit and project.run
so here we need to call appropriate method based on editor_flag. since editor_flag is a string, we can handle it with if-else
```gdscript
if editor_flag == '-e':
   item.edit()
if editor_flag == '-r':
   item.run()
```
or use some cool polymorphic stuff

```gdscript
editor_flag.call(item)
```
where editor flag is Callable with example definition:
```gdscript
func (item): item.edit()
func (item): item.run()
```

Reply:
fixed